fv3core/fv3core/stencils/c_sw.py

```diff
@@ -588,14 +673,15 @@ def __call__(
             w: vertical velocity (in)
             uc: C-grid x-velocity (inout)
             vc: C-grid y-velocity (inout)
-            ua: A-grid x-velocity (in)
-            va: A-grid y-velocity (in)
-            ut: u * dx (inout)
-            vt: v * dy (inout)
-            divgd: D-grid horizontal divergence (inout)
-            omga: Vertical pressure velocity (inout)
+            ua: A-grid x-velocity (inout)
+            va: A-grid y-velocity (inout)
+            ut: u * dx (out)
+            vt: v * dy (out)
+            divgd: D-grid horizontal divergence (out)
+            omga: Vertical pressure velocity (out)
             dt2: Half a model timestep in seconds (in)
```

Comment:
The layouts of these are inconsistent with the other in/out documentation

Reply:
Done.